src/Fields/Link.php

```diff
@@ -13,7 +13,10 @@ class Link extends BaseField
 
     public function build(array $parameter = [])
     {
-        $default = [ 'return_format' => 'array' ];
+        $default = [
```

Comment:
@pascalotti can't we add this default to the BaseField directly? It would be then there for each field by default.

https://github.com/towa-digital/towa-acf-fields/blob/a0a110b99e75ff7f0832e40a24b8235589ea84c6/src/BaseField.php#L49

something like:

```
$defaults = [
            'key'   => $this->get_key(),
            'label' => $this->get_label(),
            'name'  => $this->get_name(),
            'type'  => $this->type,
           'wpml_cf_preferences' => 2,
        ];
```

and maybe let us add a comment or introduce a constant for the "2" so we now what that represents

Reply:
ah this is also valid for the other changes you made

Reply:
pr updated.
I also tested this branch with the current version of the WP-Boilerplate.
All available Section Modules work perfectly with this solution. 🥇